zzzz_modular_occulus/code/modules/mob/living/simple_animal/hostile/siren/intruder.dm

```diff
@@ -0,0 +1,88 @@
+/mob/living/simple_animal/hostile/siren/intruder
+	name = "intruder"
+	desc = "A floating mechanical drone. It can be seen putting off sparks occasionally"
+	icon = 'zzzz_modular_occulus/icons/mob/siren/general.dmi'
+	icon_state = "intruder"
+	icon_living = "intruder"
+	icon_aggro = "intruder"
+	icon_dead = "sixleg-dead"
+	icon_gib = "syndicate_gib"
+	mouse_opacity = 2
+	move_to_delay = 8
+	vision_range = 5
+	aggro_vision_range = 11
+	agony_coefficient = 0.8
+	ranged = 1
```

Comment:
Based on elsewhere, this seems to be meant as a boolean. Use TRUE for 1 and FALSE for 0